NSS-Day-Cohort-49/tabloid-full-stack-spicy-sweet-chilis
# Pr create comment

#### Changes Made
1. Added file ability to create a comment.
?
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout pr-create-comment
    ```
2. Open a new Terminal tab (?T) and navigate to the server directory.
3. Test app functionality.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
